network/benchmarks/netperf/launch.go

```diff
@@ -344,14 +350,23 @@ func processCsvData(csvData *string) bool {
 	outputFilePrefix := fmt.Sprintf("%s-%s_%s.", testNamespace, tag, t.Format("20060102150405"))
 	fmt.Printf("Test concluded - CSV raw data written to %s/%scsv\n", outputFileDirectory, outputFilePrefix)
 	if _, err := os.Stat(outputFileDirectory); os.IsNotExist(err) {
-		os.Mkdir(outputFileDirectory, 0766)
+		err := os.Mkdir(outputFileDirectory, 0766)
+		if err != nil {
+			fmt.Printf("Error creating directory", err)
```

Comment:
Printf requires format, so somethig line:
```
fmt.Printf("Error creating directory: %v", err)
```

Though the patter used in this file seem to be to use Println, so use Println for consistency.

Reply:
done